dev/DynamicDependency/M.AM.DD.CreatePackageDependencyOptions.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "Microsoft.ApplicationModel.DynamicDependency.CreatePackageDependencyOptions.g.h"
+
+namespace winrt::Microsoft::ApplicationModel::DynamicDependency::implementation
+{
+    struct CreatePackageDependencyOptions : CreatePackageDependencyOptionsT<CreatePackageDependencyOptions>
+    {
+        CreatePackageDependencyOptions() = default;
+
+        Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures Architectures();
```

Comment:
Do you need these prefixes? You're already in the `winrt::Microsoft::ApplicationModel::DynamicDependency` namespace here.

Reply:
Good point. Old habits die hard and I was initially worried about ambiguities, but not a problem now

Reply:
See comment elsewhere about namespace aliasing.

Reply:
Hmmm. I don't I see a comment elsewhere. Are you suggesting e.g.

namespace Mdd
{
using namespace ::Microsoft::ApplicationModel::DynamicDependency;
}

and then changing
```
::Microsoft::ApplicationModel::DynamicDependency::PackageDependency GetFromId(hstring const& id);
```
to
```
Mdd::PackageDependency GetFromId(hstring const& id);
```

?

Reply:
Neat trick. Thanks